components/txn_types/src/types.rs

```diff
@@ -292,6 +312,46 @@ impl From<kvrpcpb::Mutation> for Mutation {
     }
 }
 
+// Returned by MvccTxn when extra_op is set to kvrpcpb::ExtraOp::ReadOldValue.
+// key with current ts -> (short value of the prev txn, start ts of the prev txn).
+// the value is None when the mutation is `Insert`.
```

Comment:
Could you clarify the comment, there are two Options which could be None. (Even better if we can get rid of one :-) )